server/src/test/java/org/elasticsearch/action/admin/indices/rollover/RolloverIT.java

```diff
@@ -84,6 +86,9 @@ public void testRollover() throws Exception {
         assertFalse(oldIndex.getAliases().containsKey("test_alias"));
         final IndexMetaData newIndex = state.metaData().index("test_index-000003");
         assertTrue(newIndex.getAliases().containsKey("test_alias"));
+        assertThat(oldIndex.getRolloverInfos().size(), equalTo(1));
+        assertThat(oldIndex.getRolloverInfos().get("test_alias").getAlias(), equalTo("test_alias"));
+        assertThat(oldIndex.getRolloverInfos().get("test_alias").getMetConditions(), is(empty()));
```

Comment:
I think we should make an assertion on the `getTime` too. I see a way to do this: take the clock before you submit the request, then assert the value of `getTime` is more than that?

Reply:
true, I guess I was trying avoid timing issues since the system time was being finicky. the move to threadPool makes things more consistent in their world view! thanks